src/msg/xio/XioConnection.cc

```diff
@@ -160,7 +159,7 @@ void XioConnection::send_keepalive_or_ack(bool ack, const utime_t *tp)
 {
   /* If con is not in READY state, we need to queue the request */
   if (cstate.session_state.read() != XioConnection::UP) {
-    pthread_spin_lock(&sp);
+    std::lock_guad<ceph::util::spinlock> lg(sp);
```

Comment:
typo?

Reply:
Yes, and that makes it worrisome that it compiled on both my local system and here... hmm...

Reply:
with-xio is set to OFF by default